### PR TITLE
Edits to Flux and to all tutorials

### PR DESCRIPTION
**Pull request by jbednar**
Should be ready to merge if it builds.

### CODE REVIEW

File: examples/tutorial/03_Alignment_and_Preprocessing.ipynb [hunk 1]
```diff
@@ -4,21 +4,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Alignment and Preprocessing"
```
Comment (jsignell) [inline examples/tutorial/03_Alignment_and_Preprocessing.ipynb]: This split is intentional for website usage. It allows the notebook to render with the `nbsite` generated title on the website, and not have it duplicated in the content.

**Comment by jsignell** [comment]
I am fixing the headers back and switching to new options style

**Comment by jsignell** [comment]
I think that is the first time that the smoke tests caught a real error for me! :)

**Comment by jbednar** [comment]
They have definitely caught real errors by me on other projects...

File: examples/tutorial/03_Alignment_and_Preprocessing.ipynb [hunk 2]
```diff
@@ -277,10 +277,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "NDVI_1988_p = ROI['1988'].hvplot(clim=(-3, 1), crs=crs, rasterize=True).relabel('1988')\n",
-    "NDVI_2017_p = ROI['2017'].hvplot(clim=(-3, 1), crs=crs, rasterize=True).relabel('2017')\n",
+    "args = dict(cmap='viridis', width=500, height=500, crs=crs, rasterize=True)\n",
```
Comment (jbednar) [inline examples/tutorial/03_Alignment_and_Preprocessing.ipynb]: Oops! One of the perils of out of order execution done when tired. :-)

**Pull request merged**